client/components/widget/data_viz/gviz/gviz-directive.js

```diff
@@ -301,6 +285,15 @@ explorer.components.widget.data_viz.gviz.gvizChart = function(
               scope.widgetConfig.queryError = response.error;
             });
           });
+          // Progress notification
+          promise.then(null, null, angular.bind(this, function(notification) {
+            if (notification == 'Started' /*WorkQueueService.NOTIFICATION.STARTED*/) {
```

Comment:
Done via goog.require - it's a module-level constant and shouldn't need to be injected.